add_to_app/multiple_flutters_module/README.md

```diff
@@ -0,0 +1,50 @@
+# multiple_flutters_module
+
+This is the module that is embedded in the `multiple_flutters_ios` and
+`multiple_flutters_android (TODO)` projects.
+
+## Getting Started
+
+For iOS instructions, see:
+[multiple_flutters_ios](../multiple_flutters_ios/README.md).
+
+For Android instructions, see:
+[multiple_flutters_android](../multiple_flutters_android/README.md).
+
+## Flutter Engine Group
+
```

Comment:
Add a brief context for why might someone want to have multiple engines (probably in the parent's README).

Reply:
Ok, rewrote it.  It's now in multiple_flutters/README.md